### ISSUE TITLE
add "Hub" to webpage headers; incorporate in Resources section of main AASHE navigation bar

### ISSUE THREAD

**Issue by monika-aashe** [AASHE/hub#238]
Based on feedback from Jessica, this must be done before the official launch:

Revisions to the header on all AASHE, STARS, Conference and Bulletin pages:
aashe  |  stars  |  conference  |  bulletin  |  hub 

Also need to make changes to the Resources section of the main AASHE navigation bar as follows:

- Resources (clicking on the main link will take you to http://hub.aashe.org/)
     - Campus Sustainability Hub (http://hub.aashe.org/)
     - Resource Center (http://www.aashe.org/resources) 
     - AASHE Publications (http://www.aashe.org/resources/publications-aashe)
     - AASHE Awards (http://www.aashe.org/about/aashe-awards)
     - International Resources (http://www.aashe.org/resources/shared-international-resources)
     - Sustainability Tracking, Assessment & Rating System (STARS) (https://stars.aashe.org/)
     - Green Gigawatt Partnerships (http://greengigawatt.org/)

Feel free to split into two tickets as needed.

**Comment by juliandsmith** [comment]
If we're going to update the headings anyway, let's make the options alphabetical, i.e.:
aashe I bulletin I conference I hub I stars

**Comment by jamstooks** [comment]
I have created [another issue](/AASHE/django-aashe-theme/issues/1) that addresses Julian's comment.

**Comment by monika-aashe** [comment]
Update: All except the following have been addressed: Resources (clicking on the main link will take you to http://hub.aashe.org/)

**Comment by juliandsmith** [comment]
And the headers associated with the Bulletin, partners directory, and campus sustainability month do not yet have Hub in them.

**Comment by jamstooks** [comment]
[x] campus sustainability month
[ ] bulletin
[ ] partners directory

**Comment by jamstooks** [comment]
- [ ] Resources (clicking on the main link will take you to http://hub.aashe.org/)

**Comment by jamstooks** [comment]
This is fixed. The CSM and SCPD sites update automatically every evening.
